tools/java/org.hl7.fhir.tools.core/src/org/hl7/fhir/tools/implementations/ember/EmberGenerator.java

```diff
@@ -0,0 +1,278 @@
+            String lower = name.toLowerCase();
+            serverWriter.write(lower + "Base := router.Path(\"/" + name + "\").Subrouter()");
+            serverWriter.newLine();
+            //serverWriter.write(lower + "Base.Methods(\"GET\").HandlerFunc(" + name + "IndexHandler)");
+            serverWriter.write(lower + "Base.Methods(\"GET\").Handler(negroni.New(append(config[\"" + name + "Index\"], negroni.HandlerFunc(" + name + "IndexHandler))...))");
+            serverWriter.newLine();
+            //serverWriter.write(lower + "Base.Methods(\"POST\").HandlerFunc(" + name + "CreateHandler)");
+            serverWriter.write(lower + "Base.Methods(\"POST\").Handler(negroni.New(append(config[\"" + name + "Create\"], negroni.HandlerFunc(" + name + "CreateHandler))...))");
+            serverWriter.newLine();
+            serverWriter.newLine();
+            serverWriter.write(lower + " := router.Path(\"/" + name + "/{id}\").Subrouter()");
+            serverWriter.newLine();
+            //serverWriter.write(lower + ".Methods(\"GET\").HandlerFunc(" + name + "ShowHandler)");
+            serverWriter.write(lower + ".Methods(\"GET\").Handler(negroni.New(append(config[\"" + name + "Show\"], negroni.HandlerFunc(" + name + "ShowHandler))...))");
+            serverWriter.newLine();
+            //serverWriter.write(lower + ".Methods(\"PUT\").HandlerFunc(" + name + "UpdateHandler)");
+            serverWriter.write(lower + ".Methods(\"PUT\").Handler(negroni.New(append(config[\"" + name + "Update\"], negroni.HandlerFunc(" + name + "UpdateHandler))...))");
+            serverWriter.newLine();
+            //serverWriter.write(lower + ".Methods(\"DELETE\").HandlerFunc(" + name + "DeleteHandler)");
+            serverWriter.write(lower + ".Methods(\"DELETE\").Handler(negroni.New(append(config[\"" + name + "Delete\"], negroni.HandlerFunc(" + name + "DeleteHandler))...))");
+            serverWriter.newLine();
+            serverWriter.newLine();
+        }
+
+        serverWriter.write("}");
+
+        serverWriter.flush();
+        serverWriter.close();
+    }
+
+    private void generateGoUtil(Set<String> resourceNames, String outputDir, STGroup templateGroup) throws IOException {
+        // Sort the resources by name just for consistent (diff-able) output
+        ArrayList<String> resourceList = new ArrayList<String>(resourceNames);
+        Collections.sort(resourceList);
+
+        ST utilTemplate = templateGroup.getInstanceOf("util.go");
+        utilTemplate.add("Resources", resourceList);
+
+        File outputFile = new File(Utilities.path(outputDir, "util.go"));
+        Writer controllerWriter = new BufferedWriter(new FileWriter(outputFile));
+        controllerWriter.write(utilTemplate.render());
+        controllerWriter.flush();
+        controllerWriter.close();
+    }
+
+    private static String dasherize(String str){
+      String[] r = str.split("(?=\\p{Upper})");
+      return "".join("-", r).toLowerCase();
```

Comment:
`join` is a static method, so you should use `String.join` rather than `"".join`